packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,15 +104,28 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
       decoration: BoxDecoration(
         border: Border.all(color: Theme.of(context).focusColor),
       ),
-      child: InspectorTree(
-        controller: summaryTreeController,
-        isSummaryTree: true,
+      child: GestureDetector(
+        child: InspectorTree(
+          controller: summaryTreeController,
+          isSummaryTree: true,
+        ),
+        onTapUp: (TapUpDetails tapUpDetails) {
+          summaryTreeController.setFocus();
+        },
       ),
     );
-    final detailsTree = InspectorTree(
-      controller: detailsTreeController,
+    final detailsTree = GestureDetector(
+      child: InspectorTree(
+        controller: detailsTreeController,
+      ),
+      onTapUp: (TapUpDetails tapUpDetails) {
+        detailsTreeController.setFocus();
+      },
     );
 
+    // Set focus on the summary tree by default
+    summaryTreeController.setFocus();
```

Comment:
Is there a better place to set default focus than in the build method?

Reply:
can we have the inspector screen track which tree was last selected? and then in didUpdateWidget, set the focus on the last selected tree?

Reply:
@jacob314 your thoughts?

Reply:
we should try to integrate with the core flutter concepts of tracking the current focus. Having our controllers have a separate definition of focus will lead to confusing ui. You can add a hook in the inspector controller to query the associated view to determine it has focus if you need that info in the controller.

Reply:
I would reach out to the flutter users chat to get some feedback on this. I hope we can  just make the summary tree be the default focus target.

Reply:
The documentation for https://api.flutter.dev/flutter/widgets/FocusScope-class.html also has some good breadcrumbs describing how Flutter intends to support tracking focus.

Reply:
Using a Focus node simplified the code a lot.